### ISSUE TITLE
Real-time scrubbing

### ISSUE THREAD

**Issue by katstiel** [wonderunit/storyboarder#1296]
I wasn't sure whether this was a bug or not, but as of now Storyboarder seems to only show the times that boards end and begin, rather than showing any particular point on the timeline. For example, if I have a board that's 2 seconds long and one that's 3 seconds long, I can only jump to the 2 and 5 second marks, not the 1, 3, or 4 second marks. 
This makes it hard to sync with audio, since I have to guess when I think the scene should change based on ear, enter that number manually, and check to see if I got it right. It would be much easier if I could just pause when I hear the auditory cue for a scene change and insert the board right there rather than guessing and checking.
I hope that makes sense? Just let me know if further clarification is needed!

**Comment by audionerd** [comment]
Have you tried Timeline Mode (View -> Toggle Boards/Timeline Mode)? Should make it easier to line up durations. You'll be able to see the audio in the timeline and drag board thumbnail edges to change board duration. Does that help?

**Comment by katstiel** [comment]
@audionerd Yeah, I have that enabled; it's just strange that the timer doesn't show the current time when playing the preview like every other video or audio program, and instead uses existing boards as a reference for the timing.
